fees/iotex/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Adapter, FetchOptions, ProtocolType } from "../../adapters/types";
+import { getTimestampAtStartOfDayUTC } from "../../utils/date";
+import { httpGet } from "../../utils/fetchURL";
+import { CHAIN } from "../../helpers/chains";
+
+const IOTEX_SUM_GAS_URL = "https://gateway1.iotex.me/analyzer/sumGasFeeIotx";
+
+const fetch = async (options: FetchOptions) => {
+  const dailyFees = await getDailyFees(options.startTimestamp);
+  const totalFees = await getTotalFees();
+
+  return {
+    dailyFees,
+    dailyRevenue: dailyFees,
+    totalFees,
+    totalRevenue: totalFees,
+  };
+};
+
+const adapter: Adapter = {
+  version: 2,
+  adapter: {
+    [CHAIN.IOTEX]: {
+      fetch,
+      start: "2021-06-22",
+      meta: {
+        hallmarks: [
```

Comment:
can you remove these hallmarks, we keep it mainly for spike/drop in tvl/other metrics, dont think it is applicable here

Reply:
@g1nt0ki thanks for reviewing it! removed hallmarks, to clarify, we can add them in DefiLlama-Adapters instead?

Reply:
atm, we dont have hallmarks for chains, but these actions have significant impact on the tvl? like FTX collapsing or LUNA collapsing had?